SwiftCheck/TestOperators.swift

```diff
@@ -117,4 +117,4 @@ public func ^||^(p1 : Testable, p2 : Testable) -> Property {
 	return disjoin(p1.property, p2.property)
 }
 
-import func XCTest.XCTFail
+@exported import XCTest
```

Comment:
It's also undocumented, so I'm not entirely sure what else it does or if this is what it's supposed to do. But hey, it works here.